MSAL/src/public/configuration/publicClientApplication/MSALCacheConfig.h

```diff
@@ -37,6 +37,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property NSString *keychainSharingGroup;
 
+/*!
+    Enable or disable keychain sharing group.
+ 
+    If enabled, keychainSharingGroup will be set to the default value,
+    which is com.microsoft.adalcache.
+ 
+    Disabling this will set the keychainSharingGroup to the
+    app's bundle ID; a unique value to this app and thus,
+    prevent sharing of the keychain with other applications.
+ */
+@property BOOL enableKeychainSharing;
```

Comment:
Why do we need this flag? It is still possible to enable/disabled keychain sharing by providing proper keychain group.

Reply:
For convenience. It reads better/easier in a documentation to say 
"to disable, set this flag to XXX" rather than what we have currently.

Reply:
nit: I feel this property could confuse developers. Could it be that 3P developer wants to enable sharing under its own group, so he would set it to something like "com.myapp" and at the same time he could miss the documentation of `enableKeychainSharing ` and set it to "Yes". I think it is better to provide only one way of setting this either by keeping "enableKeychainSharing" or  "keychainSharingGroup".

Reply:
@unpluggedk, @antrix1989, let's discuss this one tomorrow?